### PR TITLE
Update design for responsive practice

### PR DESCRIPTION
**Pull request by CBinh710**
# Design
[link](https://www.figma.com/design/k5dkI3GRzx7YKvWVmBuQsu/Responsive-Foodhut-design(clone)?node-id=0-1&t=0bbn5N4iC89IOzGD-0)

### CODE REVIEW

File: Foodhut/src/styles/sections/header.css [hunk 1]
```diff
@@ -1,81 +1,68 @@
...
   }
 
-  .wrapper-navbar-container {
-    display: var(--hidden-display);
-  }
-
-  img {
-    width: 100%;
-    height: auto;
+  .logo-company-container + .flex {
```
Comment (phuocduong-agilityio) [inline Foodhut/src/styles/sections/header.css]: Avoid to restyle common styles. Pls check all

File: Foodhut/src/styles/sections/introduction.css [hunk 2]
```diff
@@ -110,25 +117,70 @@
...
+
   .illustrate-image {
     width: 25.75rem;
     height: 26rem;
   }
 }
 
 /* Style for smartphone screen max width 428px */
-@media (max-width: 26.75rem) { /* 428px */
+@media (max-width: 428px) { /* 428px */
```
Comment (phuocduong-agilityio) [inline Foodhut/src/styles/sections/introduction.css]: Remove common in this line, no need

File: Foodhut/src/styles/sections/header.css (hunk 1 above)
Comment (CBinh710) [reply Foodhut/src/styles/sections/header.css]: i fixed it, thanks for your contribution

File: Foodhut/src/styles/sections/introduction.css (hunk 2 above)
Comment (CBinh710) [reply Foodhut/src/styles/sections/introduction.css]: I deleted it

**Pull request merged**